src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -4168,13 +4168,11 @@ void SmartScript::FillScript(SmartAIEventList e, WorldObject* obj, AreaTrigger c
 {
     if (e.empty())
     {
-#if defined(ENABLE_EXTRAS) && defined(ENABLE_EXTRA_LOGS)
```

Comment:
I think this should not be deleted as it controls debug log entries.

Reply:
I need to do this so travis can release the checks

Reply:
I don't understand, what do you mean by that?

Reply:
Ok, I understand. Parameter "at" is unused if extra logging is disabled and the compiler complains about this. But this has to be resolved in another way, not by deleting the preprocessor directives, as you will get debug messages even when disabling extra logs.

Reply:
So, I will revert to original changes.

Reply:
You could add
```(void)at;```
at the beginning of the function. This way the compiler sees that the parameter is referenced and won't complain any more.

Reply:
Thanks for the help.